golem-worker-executor-base/src/grpc.rs

```diff
@@ -1260,6 +1292,90 @@ impl<Ctx: WorkerCtx, Svcs: HasAll<Ctx> + UsesAllDeps<Ctx = Ctx> + Send + Sync +
         }
     }
 
+    async fn invoke_and_await_worker_json(
```

Comment:
Let's remove this

Reply:
It returns `JSON` result. If the user is needing only `JSON` result, I dont think we need to return `TypeAnnotatedValue` and a lager payload.

Reply:
We said we don't want the worker executor know about that JSON format at all, so I think we should not have such an invoke variant.